YAML/YAML.sublime-syntax

```diff
@@ -51,11 +51,11 @@ variables:
   c_indicator: '[-?:,\[\]{}#&*!|>''"%@`]'
   c_flow_indicator: '[\[\]{},]'
   ns_word_char: '[0-9A-Za-z\-]'
-  ns_uri_char: '(?x: %\p{XDigit}{2} | [0-9A-Za-z\-#;/?:@&=+$,_.!~*''()\[\]] )'
+  ns_uri_char: '(?x: %[0-9A-Fa-f]{2} | [0-9A-Za-z\-#;/?:@&=+$,_.!~*''()\[\]] )'
```

Comment:
Yes, I believe in .NET world it represents horizontal white space characters, though, as I said, anything using these syntax definitions should expect things to break unless they interpret the expressions the same way Onigurama does.
http://www.regular-expressions.info/refshorthand.html